packages/application-shell/src/components/user-settings-menu/user-settings-menu.js

```diff
@@ -95,7 +94,7 @@ const UserSettingsMenuBody = props => {
     [];
 
   return (
-    <Card className={styles.menu}>
```

Comment:
This `Card` component was basically useless, as the `.menu` styles were overwriting all of the card styles 🙃